hyrisecockpit/api/app/status/service.py

```diff
@@ -35,15 +41,20 @@ def get_database_status(cls) -> List[DatabaseStatus]:
         ]
 
     @classmethod
-    def get_benchmark_status(cls) -> List[BenchmarkStatus]:
+    def get_workload_tables(cls) -> List[WorkloadTablesStatus]:
         """Get get status for all benchmark data."""
         response = cls._send_message(
-            Request(header=Header(message="benchmark status"), body={})
+            Request(header=Header(message="workload tables status"), body={})
         )
-        return [
-            BenchmarkStatus(**interface)
-            for interface in response["body"]["benchmark_status"]
-        ]
+        workload_tables: List[WorkloadTablesStatus] = []
+        for database in response["body"]["workoad_tables"]:
+            workload_tables_status = [
+                TablesStatus(**status) for status in database["workoad_tables_status"]
+            ]
+            workload_tables.append(
+                WorkloadTablesStatus(database["id"], workload_tables_status)
+            )
+        return workload_tables
```

Comment:
See [PEP 202](https://www.python.org/dev/peps/pep-0202/):
> List comprehensions provide a more concise way to create lists in situations where [...] nested loops would currently be used.
```suggestion
        return [
            WorkloadTablesStatus(
                database["id"],
                [
                    TablesStatus(**status)
                    for status in database["workoad_tables_status"]
                ],
            )
            for database in response["body"]["workoad_tables"]
        ]
```

Reply:
I think in this case the classic for loop is just easier to read.